catalog/test/disease_controller_tests.js

```diff
@@ -0,0 +1,263 @@
+const expect = require('chai').expect;
+const sinon = require('sinon');
+
+const Disease = require('../models/disease');
+const diseaseController = require('../controllers/disease');
+const testUtil = require('./test_util');
+const imagesUtil = require('../util/clear-images');
+
+describe('==== Disease Controller ====', function() {
+  describe('getDiseases()', function() {
+    let status, json, res;
+    beforeEach(function() {
+      status = sinon.stub();
+      json = sinon.spy();
+      res = { json, status };
+      status.returns(res);
+      sinon.stub(Disease, 'find');
+    });
+
+    afterEach(function() {
+      sinon.restore();
+    });
+
+    it('test_return_200_on_success', async function() {
+      await diseaseController.getDiseases({}, res, () => {});
+      expect(status.calledOnce).to.be.true;
```

Comment:
Most of the time you don't need to check if an method/function has been called only once. In these cases you can safely remove them from your tests.